app/brave_generated_resources.grd

```diff
@@ -126,6 +126,9 @@
       <message name="IDS_SHOW_BRAVE_ADBLOCK" desc="The show Brave Ad Block menu in the app menu">
         Brave Ad Block
       </message>
+      <message name="IDS_SHOW_BRAVE_WEBCOMPAT_REPORTER" desc="The show Brave Webcompat Reporter menu item in the app menu">
```

Comment:
This description is quite different from the description you used for the macOS version (which IMO is better description). Could we use the same (without the 'Mac' bit of course!)